axelor-base/src/main/resources/data-init/input-config.xml

```diff
@@ -13,7 +13,9 @@
         <bind to="canExport" eval="can_export == 'x' ? 'true' : 'false'"/>
     </input>
 
-    <input file="base_template.csv" separator=";" type="com.axelor.apps.message.db.Template"  search="self.name = :name"  />
+    <input file="base_template.csv" separator=";" type="com.axelor.apps.message.db.Template"  search="self.name = :name"  >
```

Comment:
If we want to set birtTemplate in Template then birtTemplate should be imported first

Test with move below file import tags before importing Template:
<input file="base_birtTemplate.csv" separator=";" type="com.axelor.apps.base.db.BirtTemplate" search="self.importId = :importId"  />
<input file="base_birtTemplateParameter.csv" separator=";" type="com.axelor.apps.base.db.BirtTemplateParameter" search="self.importId = :importId"  />